Automattic/wp-calypso
# Form Components: ES6ify

For _Project Hubris_.

To test:
* Verify that all components in http://calypso.localhost:3000/devdocs/design/form-fields still work (watch the console for errors!)
* To be sure, also try a couple of form-laden Calypso sections, e.g. `/me`, or site settings.

The `export` shouldn't cause any trouble since there aren't any named ones (just `default` ones), so any `import`s/`require`s should continue to work.
